qgreenland/config/helpers/layers/sea_ice_age.py

```diff
@@ -0,0 +1,88 @@
+import json
+from typing import Literal
+
+from qgreenland.config.datasets.seaice import seaice_age as dataset
+from qgreenland.config.helpers.steps.build_overviews import build_overviews
+from qgreenland.config.helpers.steps.warp_and_cut import warp_and_cut
+from qgreenland.config.project import project
+from qgreenland.constants import CONFIG_DIR
+from qgreenland.models.config.layer import ConfigLayer, ConfigLayerInput
+from qgreenland.models.config.step import ConfigLayerCommandStep
+
+
+def _get_layer_params():
+    params_fp = CONFIG_DIR / 'helpers/layers/sea_ice_age_params.json'
+    with open(params_fp, 'r') as f:
+        return json.loads(f.read())
+
+    # TODO: check if list of years in layer params json file matches the list of
```

Comment:
TODO: we decided not to do this. Provide some info about running the script instead.